src/plugins/nativestorage.ts

```diff
@@ -2,8 +2,8 @@ import { Cordova, Plugin } from './plugin';
 
 
 /**
- * @name Native Storage
- * @description
+ * @name NativeStorage
```

Comment:
I'm the developer of the NativeStorage plugin, and I intended to use it as a whole to refer to LocalStorage.